packages/caver-transaction/src/transactionTypes/valueTransferMemo/feeDelegatedValueTransferMemoWithRatio.js

```diff
@@ -53,6 +53,18 @@ function _decode(rlpEncoded) {
  * @class
  */
 class FeeDelegatedValueTransferMemoWithRatio extends AbstractFeeDelegatedWithRatioTransaction {
+    /**
+     * Creates a fee delegated value transfer with ratio transaction.
+     * @method create
+     * @param {object|string} createTxObj - The parameters to create a FeeDelegatedValueTransferWithRatio transaction. This can be an object defining transaction information, or it can be an RLP-encoded string.
```

Comment:
```suggestion
     * @param {object|string} createTxObj - The parameters to create a FeeDelegatedValueTransferMemoWithRatio transaction. This can be an object defining transaction information, or it can be an RLP-encoded string.
```